Corrected error in Chapter 3 - Waveform Displays (2.2.x) (#221)
mixxxdj/manual: source/chapters/user_interface.rst

   Mixxx default skin (Latenight) - Parallel waveforms

Depending on the skin, Mixxx displays either parallel waveforms (default) or
separate waveforms. You can select your preferred appearance by clicking the 

> gear icon in the upper right corner. Then under Skin Settings, click/toggle 
> the Parallel Waveforms option to choose between parallel or separate waveform.

Reviewer: Let's use the [`:menuselection:` role](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-menuselection) for this and remove the repetition.

```suggestion
gear icon in the upper right corner, then toggling the :menuselection:`Skin Settings --> Parallel Waveforms` option.
```

Author: alright. let me implement this and Ronso's change. Thank you.

> separate waveforms. You can select your preferred appearance by clicking the 
> gear icon in the upper right corner. Then under Skin Settings, click/toggle 
> the Parallel Waveforms option to choose between parallel or separate waveform.

Reviewer 2: ```suggestion
separate waveforms. LateNight and Tango skins only have resizable paralel waveforms, Shade skin provides only separate waveforms. In Deere skin you can select your preferred appearance by clicking the 
gear icon in the upper right corner, then toggling the :menuselection:`Skin Settings --> Parallel Waveforms` option.

```

Author: Alright. Let me implement this. Thank you :).
